gapis/gfxapi/vulkan/vulkan.api

```diff
@@ -4607,6 +4607,11 @@ cmd VkResult vkResetDescriptorPool(
     VkDevice                   device,
     VkDescriptorPool           descriptorPool,
     VkDescriptorPoolResetFlags flags) {
+  sets := DescriptorPools[descriptorPool].DescriptorSets
```

Comment:
Interesting.
In CPP side, it is a copy, so no problem.
In Golang side, it is not a copy, however, it is **safe** to remove item while iterating!
https://stackoverflow.com/questions/23229975/is-it-safe-to-remove-selected-keys-from-golang-map-within-a-range-loop